### DISCUSSION TITLE
Neon binding for javascript

### DISCUSSION THREAD

**Discussion by killme2008** [GreptimeTeam/greptimedb#3816] [Ideas]
### What type of enhancement is this?

User experience, Other

### What does the enhancement do?

We can wrap the GreptimeDB by https://neon-bindings.com/, and provide the time-series database capability to javascript world.

**Comment by tisonkun** [comment]
> wrap the GreptimeDB

@killme2008 Does it mean to wrap the whole server? If so, only standalone or ...

Or wrap the client for accessing greptime server from nodejs.

**Comment by killme2008** [comment]
> > wrap the GreptimeDB
> 
> @killme2008 Does it mean to wrap the whole server? If so, only standalone or ...
> 
> Or wrap the client for accessing greptime server from nodejs.

The goal is to wrap the whole server as an embedded time series database for javascript.

**Comment by tisonkun** [comment]
Move to Ideas until we have some concrete proposal to proceed.
